rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -42,25 +42,46 @@ def _update_history_events_schema(write_cursor: 'DBCursor', conn: 'DBConnection'
 
         for entry in read_cursor:
             if entry[11] is None:
-                new_entries.append(entry[:11] + ('none',) + entry[12:])
+                new_entries.append(entry[:11] + ('none',))
             else:
-                new_entries.append(entry)
+                new_entries.append(entry[:12])
 
-    write_cursor.executemany('INSERT INTO history_events_copy VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ,? ,? ,?)', new_entries)  # noqa: E501
+    write_cursor.executemany('INSERT INTO history_events_copy VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', new_entries)  # noqa: E501
```

Comment:
As I said before I would copy only the non evm events since they would need to be redecoded (the evm ones) that means modifying other tables too. If you want I believe is fair to leave that as a TODO for a smaller PR

Reply:
Yes, forgot about this thing. Makes sense